externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/KyuubiSparkUtil.scala

```diff
@@ -52,7 +59,49 @@ object KyuubiSparkUtil extends Logging {
     }
   }
 
+  def getInitializeSql(sparkSession: SparkSession, config: Seq[String]): Seq[String] = {
+
+    def getSqlTextFromFile(path: Option[String]): Seq[String] = {
+      path.map(filePath => {
+        var reader: BufferedReader = null
+        try {
+          info(s"========>load init sql from file: ${filePath}")
+          val engineInitSqlFilePath = new Path(filePath)
+          val hadoopConf = sparkSession.sparkContext.hadoopConfiguration
+          val fs: FileSystem = engineInitSqlFilePath.getFileSystem(hadoopConf)
+          if (fs.exists(engineInitSqlFilePath)) {
+            val sqlInputStream: FSDataInputStream = fs.open(engineInitSqlFilePath)
+            reader = new BufferedReader(new InputStreamReader(sqlInputStream))
+            // Filter out the sql using '--' comment
+            val sqlText = ListBuffer[String]()
+            Stream.continually(reader.readLine())
+              .takeWhile(null != _).filter(item => !item.startsWith("--") && item.length > 0)
+              .map(item => item.replace(";", "")).foreach(item => sqlText.append(item))
```

Comment:
The implementation looks fragile ... have you considered 
```
select regexp_replace(html, '\\\\n|\\\\\t|\\\\\r', ';')
```

Reply:
OK, I will modify it